src/main/java/org/myrobotlab/serial/Port.java

```diff
@@ -147,8 +147,10 @@ public void run() {
         }
       }
       log.info("{} no longer listening - last byte {} ", portName, newByte);
-    } catch (Exception e) {
-      log.error("port reading thread threw", e);
+    } catch (InterruptedException e) {
+      log.info("port {} interrupted - stopping listener", portName);

```

Comment:
not logging the exception stack trace here.  it should.

Reply:
its a specific exception from an interrupt - it was intentional, not an error - the stack trace is not needed